### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by SpliiT**

### CODE REVIEW

File: components/ImportFile.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <template #loader>
+      <v-progress-circular indeterminate size="20" color="white" width="3" />
+    </template>
+  </v-btn>
+  <v-btn variant="text" @click="$emit('decrement_step')"> Cancel </v-btn>
+</template>
+
+<script setup>
+import schemas from "@geode/opengeodeweb-viewer/schemas.json";
+import schema from "@/components/importfile.json";
```
Comment (BotellaA) [inline components/ImportFile.vue:13]: It should be imported from OGW-Back

File: geodeapp_back.spec
Comment (BotellaA) [inline geodeapp_back.spec:1]: to remove

File: geodeapp_viewer.spec
Comment (BotellaA) [inline geodeapp_viewer.spec:1]: to remove

File: lock/ping.txt
Comment (BotellaA) [inline lock/ping.txt:1]: to remove, maybe add lock folder in the gitignore

File: package.json [hunk 2]
```diff
@@ -1,21 +1,16 @@
 {
   "name": "vease",
-  "version": "0.0.0-semantically-released",
+  "version": "0.0.1",
```
Comment (BotellaA) [inline package.json:3]: sure? why this change?

File: package.json [hunk 3]
```diff
@@ -1,21 +1,16 @@
...
   "description": "Basic app",
   "main": "dist-electron/main.js",
   "scripts": {
     "dev": "nuxi dev",
     "generate": "nuxt generate",
-    "install_back": "node node_scripts/install_back.js",
```
Comment (BotellaA) [inline package.json:9]: @JulienChampagnol are you ok with these changes?

File: pages/index.vue [hunk 4]
```diff
@@ -9,15 +9,10 @@
 </template>
 
 <script setup>
-const infra_store = use_infra_store();
-const geode_store = use_geode_store();
 const viewer_store = use_viewer_store();
-
-onMounted(async () => {
-  await infra_store.create_connexion();
-  geode_store.ping_task();
```
Comment (BotellaA) [inline pages/index.vue:18]: Why these have been removed?

File: pages/supperposition.vue [hunk 5]
```diff
@@ -1 +1,140 @@
-<template></template>
+<template>
+  <Stepper />
+  <!-- <FileSelector /> -->
```
Comment (BotellaA) [inline pages/supperposition.vue:3]:
```suggestion
```

File: pages/supperposition.vue [hunk 6]
```diff
@@ -1 +1,140 @@
...
+</template>
+<script setup>
```
Comment (BotellaA) [inline pages/supperposition.vue:5]:
```suggestion

<script setup>
```

File: pages/supperposition.vue [hunk 7]
```diff
@@ -1 +1,140 @@
...
+import _ from "lodash";
+
+import FileSelector from "@geode/opengeodeweb-front/components/FileSelector.vue";
+import MissingFilesSelector from "@geode/opengeodeweb-front/components/MissingFilesSelector.vue";
+import ObjectSelector from "@geode/opengeodeweb-front/components/ObjectSelector.vue";
+import ImportFile from "@/components/ImportFile.vue";
+
+// import versions_schema from "@/components/Tools/CrsConverter/PackagesVersions.json";
```
Comment (BotellaA) [inline pages/supperposition.vue:13]:
```suggestion
```

File: plugins/vuetify.js
Comment (BotellaA) [inline plugins/vuetify.js:1]: is this file used?

**Review by BotellaA — APPROVED** [review]

**Review by JulienChampagnol — APPROVED** [review]
